### ISSUE TITLE
Error: `cannot load such file --fontist/manifest/install (LoadError)`

### ISSUE THREAD

**Issue by Intelligent2013** [metanorma/metanorma#243]
Error occurs when I run `bundle exec metanorma -t iso -x presentation sources/document.adoc` (https://github.com/metanorma/iso-iec-smart-terms):
```
Traceback (most recent call last):
        9: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/bin/metanorma:23:in `<main>'
        8: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/bin/metanorma:23:in `load'
        7: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/metanorma-cli-1.5.13/exe/metanorma:24:in `<top (required)>'
        6: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/metanorma-cli-1.5.13/exe/metanorma:24:in `require'
        5: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/metanorma-cli-1.5.13/lib/metanorma/cli.rb:3:in `<top (required)>'
        4: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/metanorma-cli-1.5.13/lib/metanorma/cli.rb:3:in `require'
        3: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/metanorma-1.4.7/lib/metanorma.rb:11:in `<top (required)>'
        2: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/metanorma-1.4.7/lib/metanorma.rb:11:in `require'
        1: from C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/metanorma-1.4.7/lib/metanorma/compile.rb:6:in `<top (required)>'
C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/metanorma-1.4.7/lib/metanorma/compile.rb:6:in `require': cannot load such file --fontist/manifest/install (LoadError)
```

Gemfile.lock and previous working version Gemfile.lock.old:  [Gemfile.zip](https://github.com/metanorma/metanorma/files/8506807/Gemfile.zip)

**Comment by opoudjis** [comment]
That should be resolved in upcoming release, there was an issue in fontist being downgraded because of a library incompatibility, which should now be resolved. I see the new Gemfile is using fontist 1.2, the old 1.14.

Updated metanorma just a couple of hours ago, try updating now?

**Comment by Intelligent2013** [comment]
```

>bundle update
Fetching gem metadata from https://rubygems.org/...............
Fetching gem metadata from https://rubygems.org/.
Resolving dependencies..........................................................................................................................................
..............................
Using rake 13.0.6
Using concurrent-ruby 1.1.10
Using i18n 1.10.0
Using minitest 5.15.0
Using tzinfo 2.0.4
Using activesupport 7.0.2.3
Using public_suffix 4.0.7
Using addressable 2.8.0
Using faraday-em_http 1.0.0
Using faraday-em_synchrony 1.0.0
Using faraday-excon 1.1.0
Using faraday-httpclient 1.0.1
Using faraday-net_http 1.0.1
Using faraday-net_http_persistent 1.2.0
Using faraday-patron 1.0.0
Using faraday-rack 1.0.0
Using multipart-post 2.1.1
Using ruby2_keywords 0.0.5
Using faraday 1.8.0
Using multi_json 1.15.0
Using connection_pool 2.2.5
Using net-http-persistent 4.0.1
Using algolia 2.1.1
Using amazing_print 1.4.0
Using asciidoctor 2.0.17
Using asciimath 2.0.3
Using htmlentities 4.3.4
Using racc 1.6.0
Using nokogiri 1.12.5 (x64-mingw32)
Using rsec 1.0.0
Using asciimath2unitsml 0.4.2
Using bcp47 0.3.3
Using latex-decode 0.3.2
Using bibtex-ruby 6.0.0
Using builder 3.2.4
Using bundler 2.1.4
Using camertron-eprun 1.1.1
Using cldr-plurals-runtime-rb 1.1.0
Using cliver 0.3.2
Using cnccs 0.1.6
Using coderay 1.1.3
Using css_parser 1.11.0
Using date 3.2.2
Using descriptive_statistics 2.5.1
Using unf_ext 0.0.8.1 (x64-mingw32)
Using unf 0.1.4
Using domain_name 0.5.20190701
Using down 5.3.1
Using hamster 3.0.0
Using link_header 0.0.8
Using rdf 3.1.15
Using scanf 1.0.0
Using sxp 1.1.0
Using unicode-types 1.7.0
Using ebnf 2.2.1
Using ffi 1.15.5 (x64-mingw32)
Using emf2svg 1.3.0 (x64-mingw32)
Using equivalent-xml 0.6.0
Using thor 1.2.1
Using expressir 1.2.3 (x64-mingw32)
Using faraday_middleware 1.2.0
Using ffi-compiler2 2.0.1
Using libmspack 0.1.0
Using rubyzip 2.3.2
Using fontist 1.2.1
Using gb-agencies 0.0.7
Using rchardet 1.8.0
Using git 1.11.0
Using graphql 1.13.6
Using graphql-client 0.16.0
Using temple 0.8.2
Using tilt 2.0.10
Using haml 5.2.2
Using hashie 4.1.0
Using hollaback 0.1.1
Using image_size 3.0.1
Using mime-types-data 3.2022.0105
Using mime-types 3.4.1
Using plane1converter 0.0.1
Using thread_safe 0.3.6
Using uuidtools 2.2.0
Using html2doc 1.3.1
Using http-cookie 1.0.4
Using iev 0.3.0
Using iso-639 0.3.5
Using iso639 1.3.2
Using marcel 1.0.2
Using sterile 1.0.23
Using metanorma-utils 1.2.8
Using isodoc-i18n 1.0.2
Using liquid 4.0.3
Using mathml2asciimath 0.0.13
Using mdb 0.5.0
Using timeout 0.2.0
Using net-protocol 0.1.3
Using time 0.2.0
Using net-ftp 0.1.3
Using relaton-bib 1.11.1
Using relaton-3gpp 1.11.0
Using net-http-digest_auth 1.4.1
Using rubyntlm 0.6.3
Using webrick 1.7.0
Using webrobots 0.1.2
Using mechanize 2.8.4
Using serrano 1.0.0
Using relaton-bipm 1.11.0
Using isoics 0.1.11
Using relaton-iso-bib 1.11.0
Using relaton-bsi 1.11.0
Using relaton-calconnect 1.11.0
Using relaton-cen 1.11.0
Using relaton-cie 1.11.0
Using relaton-ecma 1.11.0
Using relaton-gb 1.11.0
Using relaton-iana 1.11.0
Using relaton-iec 1.11.0
Using relaton-ieee 1.11.0
Using relaton-ietf 1.11.0
Using relaton-iho 1.11.0
Using relaton-iso 1.11.0
Using relaton-itu 1.11.0
Using relaton-nist 1.11.0
Using relaton-oasis 1.11.0
Using relaton-ogc 1.11.0
Using relaton-omg 1.11.2
Using relaton-un 1.11.0
Using json-canonicalization 0.3.0
Using rack 2.2.3
Using json-ld 3.1.10
Using json-ld-preloaded 3.1.6
Using rexml 3.2.5
Using rdf-xsd 3.1.1
Using logger 1.5.1
Using rdf-aggregate-repo 3.1.0
Using sparql-client 3.1.2
Using sparql 3.1.8
Using ld-patch 3.1.3
Using rdf-isomorphic 3.1.1
Using rdf-json 3.1.0
Using rdf-vocab 3.1.14
Using rdf-rdfa 3.1.3
Using rdf-microdata 3.1.4
Using rdf-n3 3.1.2
Using rdf-normalize 0.4.0
Using rdf-ordered-repo 3.1.1
Using rdf-rdfxml 3.1.1
Using rdf-reasoner 0.7.2
Using rdf-tabular 3.1.1
Using rdf-turtle 3.1.3
Using rdf-trig 3.1.2
Using rdf-trix 3.1.1
Using shacl 0.1.1
Using shex 0.6.4
Using linkeddata 3.1.6
Using relaton-w3c 1.11.0
Using relaton 1.11.0
Using thor-hollaback 0.2.1
Using relaton-cli 1.9.5
Using roman-numerals 0.3.0
Using twitter_cldr 6.11.2
Using isodoc 2.0.8.1
Using ox 2.14.11
Using latexmath 0.1.5
Using lutaml-express 0.2.1
Using parslet 2.0.0
Using ruby-graphviz 1.2.5
Using lutaml-uml 0.4.3
Using lutaml-xmi 0.2.3
Using lutaml 0.7.4
Using mn2pdf 1.41.1
Using method_source 1.0.0
Using pry 0.14.1
Using metanorma 1.4.7
Using metanorma-plugin-datastruct 0.2.0
Using premailer 1.11.1
Using nokogiri-styles 0.1.2
Using reverse_markdown 1.4.0
Using sys-proctable 1.2.6
Using word-to-markdown 1.1.8
Using reverse_adoc 0.3.5
Using metanorma-plugin-lutaml 0.4.15
Using relaton-iev 1.1.3
Using optout 0.0.2
Using ruby-jing 0.0.2
Using unicode2latex 0.0.6
Using metanorma-standoc 2.0.7
Using metanorma-generic 2.0.4
Using mnconvert 1.16.0
Using tokenizer 0.3.0
Using metanorma-iso 2.0.8
Using metanorma-bipm 2.0.8
Using metanorma-cc 2.0.8
Using metanorma-csa 2.0.8
Using metanorma-iec 2.0.8
Using metanorma-ietf-data 0.2.0
Using metanorma-ietf 3.0.5
Using metanorma-iho 0.5.8
Using tzinfo-data 1.2022.1
Using metanorma-itu 2.0.8
Using metanorma-m3aawg 2.0.8
Using metanorma-ogc 2.0.8
Using metanorma-un 0.8.8
Using socksify 1.7.1
Using metanorma-cli 1.5.13
Bundle updated!
```

Error occurs again...

Gemfile.lock is identical in .zip above.

**Comment by Intelligent2013** [comment]
Solved after upgrade to metanorma 1.4.7.1.
